consensus/event_loop_test.go

```diff
@@ -691,7 +691,7 @@ func TestVoteQCConstructionSignal(t *testing.T) {
 		select {
 		case blockConstructionSignal = <-fc.Events:
 			// Do nothing
-		case <-time.After(10 * time.Millisecond):
+		case <-time.After(100 * time.Millisecond):
```

Comment:
This change is part of a PR lower down in the stack. Remember to run `gt rs --restack` locally and resubmit this PR before merging